stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -82,15 +84,28 @@ public fun Messages(
     onMessagesStartReached: () -> Unit,
     onLastVisibleMessageChanged: (Message) -> Unit,
     onScrolledToBottom: () -> Unit,
+    onMessagesEndReached: (String) -> Unit,
+    onScrollToBottom: () -> Unit,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(vertical = 16.dp),
-    helperContent: @Composable BoxScope.() -> Unit = { DefaultMessagesHelperContent(messagesState, lazyListState) },
+    helperContent: @Composable BoxScope.() -> Unit = {
+        DefaultMessagesHelperContent(
+            messagesState = messagesState,
+            lazyListState = lazyListState,
+            scrollToBottom = onScrollToBottom
+        )
+    },
     loadingMoreContent: @Composable () -> Unit = { DefaultMessagesLoadingMoreIndicator() },
     itemContent: @Composable (MessageListItemState) -> Unit,
 ) {
-    val (_, isLoadingMore, endOfMessages, messages) = messagesState
+    val messages = messagesState.messageItems
+    val endOfMessages = messagesState.endOfMessages
+    val startOfMessages = messagesState.startOfMessages
+    val isLoadingMoreNewMessages = messagesState.isLoadingMoreNewMessages
+    val isLoadingMoreOldMessages = messagesState.isLoadingMoreOldMessages
+    val scrollToPositionState = messagesState.scrollToPositionState
 
-    var parentSize by remember { mutableStateOf(IntSize(0, 0)) }
+    var parentSize = remember { IntSize(0, 0) }
```

Comment:
Why isn't this wrapped with a state?

Reply:
No need to wrap it with state, it is not observed anywhere, only passed so that we can calculate the offset. There is no need for it to be a state.